src/pages/derivx/_accounts.tsx

```diff
@@ -80,7 +91,7 @@ const StyledLinkButton = styled(LinkButton)`
     width: auto;
     margin: auto;
 `
-const StyledHeader = styled(Header)`
+const StyledHeader = styled(Header)<StyledHeaderProps>`
```

Comment:
I'm seeing props `mobile_font_size` and `mobile_margin` but type declaration as:

```
type StyledHeaderProps = {
    'font-size': string
    margin: string
}
```

Reply:
Ohh my fail...
Thanks!